docs/project/get-help.md

```diff
@@ -67,22 +67,27 @@ community members and developers.
 </table>
 
 
-## IRC Quickstart
+# IRC Quickstart 
 
-IRC can also be overwhelming for new users. This quickstart shows you 
-the easiest way to connect to IRC. 
+The following instructions show you how to register with two web based IRC
+tools. Pick one or find another. While these instructions are only for two IRC
+Web Clients there are many IRC Clients available on most platforms.
```

Comment:
web clients lowercase as they are general
comma after clients
"Pick one or find another." really makes more sense as the concluding sentence... Use one illustrated here or find another....